### DISCUSSION TITLE
Add a `bumpVersion` type that is based on the version changes

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#32422] [Suggest an Idea]
### Tell us more.

Maybe call it type `AUTO` and it bumps the version part depending on the changes. For example, if the changes only contain patch updates, then patch is bumped. If the changes contain at least 1 minor updates, then the minor version is bumped (for example, 2 minors and 5 patches, bumps minor). Same for major.

**Comment by jplitza** [comment]
I'd like a bit more config, like specifying a range of allowed version components ("Always update at least minor version"). Otherwise, THIS!

It would probably be possible to achieve this functionality by combining Renovate with [semantic-release](https://github.com/semantic-release/semantic-release): Simply define different `semanticCommitType` for different update types and let semantic-release handle the version bump. But that's yet another tool when the functionality is mostly there, in Renovate, already.
